### DISCUSSION TITLE
[vote ended on 2023-09-27] Adjust Ansible 9 roadmap to better place alpha releases

### DISCUSSION THREAD

**Discussion by felixfontein** [ansible-community/community-topics#274] [Votes done]
Right now we have an alpha release planned one week before ansible-core 2.16rc1, and another one a week afterwards. It makes more sense to have one right after the rc1 release.

Vote +1 to merge the roadmap updates from https://github.com/ansible/ansible-documentation/pull/404, or -1 to keep the current roadmap.

If you want to discuss this, please use https://github.com/ansible-community/community-topics/issues/275. Use this GitHub discussion only for votes.

**Comment by briantist** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by acozine** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by gundalow** [comment]
+1

**Comment by anweshadas** [comment]
+1

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by cybette** [comment]
+1
